frontend/tsconfig.json

```diff
@@ -7,6 +7,7 @@
     "importHelpers": true,
     "moduleResolution": "node",
     "experimentalDecorators": true,
+    "noImplicitAny": false,
```

Comment:
We should set this true, otherwise it will make the typechecking i.e. the reason why we use typescript in the first place obsolete. 
If i understand this option correctly it allows something like 
```function myfunc(par)```
without specifying a type for par, right?